addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -631,6 +633,7 @@ async function toInline($editable, cssRules, $iframe) {
  * @param {Element} editable
  */
 async function flattenBackgroundImages(editable) {
+    await loadJS('/web_editor/static/lib/html2canvas.js');
```

Comment:
Just so you know, this library is also used in `point_of_sale` :)

Reply:
yes. but they already have it in their assets, so no change is needed